packages/jobs/lib/env.ts

```diff
@@ -0,0 +1,3 @@
+import { ENVS, parseEnvs } from '@nangohq/utils';
+
+export const envs = parseEnvs(ENVS.required({ ORCHESTRATOR_SERVICE_URL: true }));
```

Comment:
should be in .env.example (at least commented) otherwise new dev install won't work